packages/tools/src/utilities/getAnnotationNearPoint.ts

```diff
@@ -102,9 +102,15 @@ function findAnnotationNearPointByTool(
     enabledElement.viewport.element,
     (tool.constructor as typeof BaseTool).toolName
   );
+  const currentId = enabledElement.viewport?.getCurrentImageId();
```

Comment:
Interesting bug, but this only exists for StackViewport, VolumeViewport don't have `getCurrentImageId`.  We are using `element` here for `isPointNearTool`, I don't understand why the wrong annotation is selected?